database/manual-updates/2021-06-16_1-add-personal-relationship-types.md

```diff
@@ -0,0 +1,101 @@
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('grandparent');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('parent');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('stepParent');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('auntUncle');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('child');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('stepChild');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('unbornChild');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('partner');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('exPartner');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('sibling');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('halfSibling');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('stepSibling');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('unbornSibling');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('spouse');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('cousin');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('nieceNephew');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('fosterCarer');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('friend');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('other');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('exSpouse');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('parentOfUnbornChild');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('siblingOfUnbornChild');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('fosterCarerSupportCarer');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('privateFosterCarer');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('privateFosterChild');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('fosterChild');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('supportCarerFosterCarer');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('neighbour');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('inContactWith');
+insert into dbo.sccv_personal_relationship_type (description) VALUES ('acquaintance');
+
+-- add inverse personal relationship types
+update dbo.sccv_personal_relationship_type set inverse_type_id = (select id from dbo.sccv_personal_relationship_type where description ilike 'greatGrandchild') where id = (select id from dbo.sccv_personal_relationship_type where description ilike 'greatGrandparent');
```

Comment:
would it be worth using '=' instead of 'ilike' for matching the strings? For safety

Reply:
Yup, will make that change!